src/github3/repos/commit.py

```diff
@@ -3,9 +3,11 @@
 from __future__ import unicode_literals
 
 from . import status
-from .. import git, models, users
+from .. import checks, git, models, users
 from .comment import RepoComment
 
+from ..decorators import requires_auth
```

Comment:
We haven't fully moved to this elsewhere, but I'd like to import the decorators module instead of the decorators from it.

Reply:
Can do!

Reply:
done.